.github/workflows/auto-label.yml

```diff
@@ -0,0 +1,18 @@
+name: Auto-label when user responds
```

Comment:
You might want to look at this for a better approach: https://github.com/microsoft/vscode-python/blob/main/.github/workflows/triage-info-needed.yml

It could be that more than one user reports the issue. so the original commenter many not be the one who responds with the details.

Reply:
Thanks I will take a look😊